### ISSUE TITLE
Wrap text in paragraphs

### ISSUE THREAD

**Issue by sharwell** [dotnet/roslyn-analyzers#4835]
### Describe the problem you are trying to solve

Text in documentation comments will not render as paragraphs unless `<para>` elements are used.

### Describe suggestions on how to achieve the rule

We can report a diagnostic for cases where a documentation element contains a blank line separating two regions of inline text:

```
/// line of text
///
/// line of text 👈 this looks like a new paragraph, but isn't
```

### Additional context

A code fix would convert the above to this:

```csharp
/// <para>line of text</para>
///
/// <para>line of text</para>
```

**Comment by Youssef1313** [comment]
I've seen some doing:

```csharp
/// line of text
/// <para />
/// line of text
```

Should this warn as well? I'm not sure.

**Comment by mavasani** [comment]
This rule would be under category "Documentation"

**Comment by sharwell** [comment]
@Youssef1313 Seeing that form in pull requests was the final straw that led me to create this proposal. 😅
